nautobot/docs/requirements.txt

```diff
@@ -1,2 +1,3 @@
-mkdocs==1.2.3
+Jinja2==3.0.3
+mkdocs==1.3.0
```

Comment:
A bit of a collision here between fixes I think - it looks like while Jinja2 3.1.0 is not compatible with mkdocs 1.2.3, mkdocs 1.2.4 and/or 1.3.0 *are* compatible with Jinja2 3.1.0, so the Jinja2 pin may not be strictly necessary here.

Also, we should be pinning Jinja2 to the same version in `docs/requirements.txt` as we are pinning in `pyproject.toml`/`poetry.lock` - so in `develop` this should really be `Jinja2==2.11.3` and we should pin `Jinja2==3.0.3` in `next` until we update `next` to Jinja2 3.1.x.